plugin/build.gradle

```diff
@@ -32,7 +32,7 @@ android {
 
 dependencies {
     compile project(':thirtyinch')
-    compile 'com.pascalwelsch.compositeandroid:activity:0.2.2'
+    compile "com.pascalwelsch.compositeandroid:activity:$supportLibraryVersion"
```

Comment:
compositeandroid introduced the supportlibrary versioning within the last week ;)